tiledb/sm/fragment/fragment_metadata.h

```diff
@@ -1222,7 +1261,7 @@ class FragmentMetadata {
   /* ********************************* */
 
   /** The storage manager. */
-  StorageManager* storage_manager_;
+  ContextResources* context_resources_;
```

Comment:
How about we change this to `resources_` so that we're consistent everywhere?

Reply:
I thought I had renamed it, thanks!